server/api.js

```diff
@@ -9,12 +9,26 @@ router.get("/", (_, res) => {
 	res.json({ message: "Hello, world!" });
 });
 
-router.post("/", (req, res) => {
-	const input = req.body.input;
+//post corrections route
+router.post("/", async (req, res) => {
+	const apiKey = process.env.OPEN_AI_SECRET_KEY;
+	const text = req.body.input; //taking the text data from inputbox
+	const configuration = new Configuration({
+		apiKey: apiKey,
+	});
+	const openai = new OpenAIApi(configuration);
 
-	// convert the input value to uppercase
-	const output = input.toUpperCase();
+	const completion = await openai.createChatCompletion({
+		model: "gpt-3.5-turbo",
+		messages: [
+			{
+				role: "user",
+				content: `correct any grammar or spelling mistakes in the sentence and offer three alternative choices : ${text}`,
```

Comment:
For now maybe just get it to return the best version of the sentence, something like the one we looked at on our call

```
`
You are a copywriter. Your job is to take some user text and make it better.
- Improve the grammar
- Make it more engaging 
- Make it as concise as possible without sacrificing clarity

The user typed the following:

"${text}"

Please re-write it, and make it better
`
```

Reply:
I think the 'alternative sentences' seems like a different feature and will require a different query?

Reply:
The exact topic I would like to discuss tomorrow.